handbook/engineering/testing.md

```diff
@@ -0,0 +1,9 @@
+# Testing
+
+This file documents how we test code at Sourcegraph. See [permissions](#permissions) for help on accessing our testing tools. 
+
+## Permissions
+
+### Percy
```

Comment:
```suggestion
## Percy
```

I think Percy would be the top level item in this document (and maybe a permissions item under it instead, but I don't think there is enough content to warrant that).